src/utils/cairoParsing.ts

```diff
@@ -13,7 +13,14 @@ export type RawCairoFunctionInfo = {
  *  @returns A list of each function information
  */
 export function parseMultipleRawCairoFunctions(rawFunctions: string): RawCairoFunctionInfo[] {
-  const functions = rawFunctions.matchAll(/func (\w+)\s*[{]?.*?[}]?\s*[(].*?[)]/gis);
+  const functions = [
+    // For cairo1
+    ...rawFunctions.matchAll(/fn (\w+)\s*[{].*?[}]/gis),
```

Comment:
Cairo 1 uses `implicits()` syntax https://github.com/starkware-libs/cairo/blob/main/corelib/src/array.cairo#L7

Reply:
Yes, updated the code. I think is safe to leave them out for now